README.md

```diff
@@ -61,13 +61,14 @@ direct:
 
 ## SelectorSyncSet Deployment
 
-In the `config.yaml` file you define a top level property `selectorSyncSet`.  Within this configuration is supported for `matchLabels`, `matchExpressions`, `matchLabelsApplyMode`, `resourceApplyMode`, and `applyBehavior`.
+In the `config.yaml` file you define a top level property `selectorSyncSet`.  Within this configuration is supported for `matchLabels`, `matchExpressions`, `matchLabelsApplyMode`, `resourceApplyMode`, `applyBehavior` and `policyComplianceType`.
```

Comment:
`policyComplianceType` should be `complianceType` and be under a new top level property.  Placing it under `selectorSyncSet` implies configuration for SSS which is not true.